src/Message.php

```diff
@@ -16,7 +16,7 @@
     public function __construct($payload, $timestamp, $encoding)
     {
         $this->payload = $payload;
-        $this->timestamp = new \DateTimeImmutable($timestamp);
+        $this->timestamp = new \DateTimeImmutable("@$timestamp");
```

Comment:
I see, so I have a propose here... Let's not use unix timestamp, we discuss this today and would be good to use ISO format, what you think about?